.gitignore

```diff
@@ -35,6 +35,8 @@ tmp/
 ### Terraform related ###
 # Local .terraform directories
 **/.terraform/*
+**/.terraform.lock.hcl
+.tfvars
```

Comment:
Usually the lock file shouldn't be ignored. I suppose you were trying to apply things in this repo directly?

Reply:
Yeah, I remember we discussed this before, but I wonder if it is still the case, since now we run terraform apply in the infra pkg. My understanding is that we only need hcl file when we want to run terraform init and apply, pkgs like lumberjack, jvs, and pmap only provide terraform module templates, we shouldn't need the hcl file?

Reply:
Discussed offline, can be ignored, closing.